classes/PrestaShopLogger.php

```diff
@@ -158,9 +158,12 @@ public static function addLog($message, $severity = 1, $errorCode = null, $objec
             $log->id_employee = (int) $idEmployee;
         }
 
-        if (!empty($objectType) && !empty($objectId)) {
-            $log->object_type = pSQL($objectType);
-            $log->object_id = (int) $objectId;
+        if (!empty($object_type)) {
+            $log->object_type = pSQL($object_type);
```

Comment:
This will never work, the variable names are incorrect vs the method's parameters:
```suggestion
        if (!empty($objectType)) {
            $log->object_type = pSQL($objectType);
```

Reply:
Sorry, I missed that.

Reply:
I think, we should not escape with `pSQL()` here, nor in `$log->message = pSQL($message);` above.
This leads to double escaping.